### PR TITLE
Feat/add electron browser to playwright

### LINKED ISSUE #825: How can I attach to electron window?
I test application which consists of standalone electron windows. Can I attach to electron window using cedeceptjs?

### PR DESCRIPTION
**Pull request by cbayer97**
## Motivation/Description of the PR
- My application gets deployed to the web and also packaged into an Electron container. Therefore, I'd like to test it in both the web and the container. Unfortunately, that is not yet possible with CodeceptJs and I dislike having to maintain two different setups. Playwright [allows to launch electron containers](https://github.com/microsoft/playwright/tree/master/packages/playwright-electron) the same way you can launch browsers (`playwright.chromium.launch(...)` vs. `playwright._electron.launch(...)`). I'd love to see this additional "browser" being added to the CodeceptJs Playwright helper and would be willing to do that if you are in favor of the idea.
- Resolves #825

Applicable helpers:

- [ ] WebDriver
- [ ] Puppeteer
- [ ] Nightmare
- [ ] REST
- [ ] FileHelper
- [ ] Appium
- [ ] Protractor
- [ ] TestCafe
- [X] Playwright

Applicable plugins:

- [ ] allure
- [ ] autoDelay
- [ ] autoLogin
- [ ] customLocator
- [ ] pauseOnFail
- [ ] puppeteerCoverage
- [ ] retryFailedStep
- [ ] screenshotOnFail
- [ ] selenoid
- [ ] stepByStepReport
- [ ] wdio

## Type of change

- [ ] :fire: Breaking changes
- [X] :rocket: New functionality
- [ ] :bug: Bug fix
- [ ] :clipboard: Documentation changes/updates
- [ ] :hotsprings: Hot fix
- [ ] :hammer: Markdown files fix - not related to source code
- [ ] :nail_care: Polish code

## Checklist:

- [ ] Tests have been added
- [ ] Documentation has been added (Run `npm run docs`)
- [ ] Lint checking (Run `npm run lint`)
- [ ] Local tests are passed (Run `npm test`)

### CODE REVIEW

File: lib/helper/Playwright.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ let defaultSelectorEnginesInitialized = false;
 
 const popupStore = new Popup();
 const consoleLogStore = new Console();
-const availableBrowsers = ['chromium', 'webkit', 'firefox'];
+const availableBrowsers = ['chromium', 'webkit', 'firefox', '_electron'];
```
Comment (DavertMik) [inline lib/helper/Playwright.js:38]:
```suggestion
const availableBrowsers = ['chromium', 'webkit', 'firefox', 'electron'];
```
Maybe let's just call it `electron`? Why not?

**Review by DavertMik — CHANGES_REQUESTED** [review]
Hey, this PR is awesome!
But it needs to be documented so everyone knew how to test Electron apps!

Could you add a section about Electron testing to a guide here https://github.com/codeceptjs/CodeceptJS/blob/3.x/docs/playwright.md

And list `electron` as an option here:
https://github.com/codeceptjs/CodeceptJS/blob/3.x/lib/helper/Playwright.js#L61

Thanks!

File: lib/helper/Playwright.js (hunk 1 above)
Comment (cbayer97) [reply lib/helper/Playwright.js]: Hey,
sorry I had a few busy weeks and it took me a while to finish this PR. Please have a look again, I have renamed the browser from `_electron` to `electron` and added some documentation as well as some other needed changes.

Comment (DavertMik) [reply lib/helper/Playwright.js]: Thanks @cbayer97 
This was a very hard week (month) for me so I just got here to look at updated PR. Everything looks good! :+1:

**Pull request merged**